portfolio/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Remove this file, it looks like it's from the linter changes.  You should do those in a separate branch, all together